FirebaseAuth/Tests/Unit/SwiftAPI.swift

```diff
@@ -583,6 +596,12 @@ class AuthAPI_hOnlyTests: XCTestCase {
       user.link(with: provider as! FederatedAuthProvider, uiDelegate: nil) { _, _ in
       }
     #endif
+    if #available(iOS 15.0, macOS 12.0, tvOS 16.0, *) {
+      user.startPasskeyEnrollment(with: "token") { _, _ in
+      }
```

Comment:
finalize variation as well for the User API.

Reply:
Added.